pox-locking/src/events.rs

```diff
@@ -419,6 +423,51 @@ pub fn synthesize_pox_event_info(
     function_name: &str,
     args: &[Value],
     response: &ResponseData,
+) -> Result<Option<Value>, ClarityError> {
+    // the first thing we do is check the current epoch. In Epochs <= 2.4,
+    //  synthesizing PoX events was an assessed cost, so event generation
+    //  must remain identical.
```

Comment:
Wow, what an oversight! The whole point of synthesizing events outside of the contract was so we could _change them_ without doing a hard fork.  Looks like that's not actually possible today?

Reply:
Agreed. Yeah, that's not possible today, but would be after 2.5 activates.